common/src/thrift/tachyon.thrift

```diff
@@ -94,6 +95,41 @@ struct Command {
   2: list<i64> mData
 }
 
+struct LineageCommand {
+  1: CommandType mCommandType
```

Comment:
I think it might make sense to make all these new thrift variables to not have "m" prefix, because the generated methods look like command.getMCommandType() and that looks strange.